document/src/vespa/document/base/fieldpath.h

```diff
@@ -70,7 +66,7 @@ class FieldPathEntry {
 
     uint32_t getIndex() const { return _lookupIndex; }
 
-    const FieldValueCP & getLookupKey() const { return _lookupKey; }
+    FieldValue & getLookupKey() const { return *_lookupKey; }
```

Comment:
Should this return `const FieldValue&` to avoid any accidental key mutations by an over-eager caller?

Reply:
Done.